M/MAGEMin/build_tarballs.jl

```diff
@@ -58,20 +60,46 @@ augment_platform_block = """
     augment_platform!(platform::Platform) = augment_mpi!(platform)
 """
 
+## These are the platforms we will build for by default, unless further
+## platforms are passed in on the command line
+#platforms = supported_platforms()
+#platforms = expand_gfortran_versions(platforms)
+
+#platforms, platform_dependencies = MPI.augment_platforms(platforms)
+
+## Avoid platforms where the MPI implementation isn't supported
+## OpenMPI
+#platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p) == "glibc"), platforms)
+## MPItrampoline
+#platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
+#1platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
+
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
-platforms = expand_gfortran_versions(platforms)
 
-platforms, platform_dependencies = MPI.augment_platforms(platforms)
+#Platform("i686", "windows"),
+platforms = expand_gfortran_versions(supported_platforms(exclude=[Platform("i686","linux"; libc="musl"),
+                                                                  Platform("i686","linux"; libc="gnu"),
+                                                                  Platform("x86_64","freebsd"),
+                                                                  Platform("armv6l","linux"; libc="musl"),
+                                                                  Platform("armv7l","linux"; libc="musl"),
+                                                                  Platform("armv7l","linux"; libc="gnu"),
+                                                                  Platform("aarch64","linux"; libc="musl")]))
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/pull/7890#discussion_r1439595948
https://github.com/JuliaPackaging/Yggdrasil/blob/48af117395188f48d361a46ea929ee7563d9c2e4/H/HDF5/build_tarballs.jl#L245

Reply:
It solved the issues, amazing thanks a lot!